packages/sources/src/components/dropdown-menu/dropdown-menu.scss

```diff
@@ -0,0 +1,211 @@
+    top: 100%;
+    z-index: $zindex-dropdown;
+  }
+
+  > .dso-primary,
+  > .dso-secondary,
+  > .dso-tertiary,
+  > .btn {
+    &::after {
+      content: "";
+      display: inline-block;
+      margin-left: $u1;
+    }
+  }
+
+  > .dso-primary,
+  > .btn-primary {
+    &::after {
+      @include di("chevron-down-wit");
+    }
+  }
+
+  > .dso-secondary,
+  > .btn-default {
+    &::after {
+      @include di("chevron-down");
+    }
+
+    &:hover,
+    &:focus {
+      &::after {
+        @include di-variant("chevron-down-wit");
+      }
+    }
+  }
+
+  > .dso-tertiary,
+  > .btn-link {
+    &::after {
+      @include di("chevron-down");
+
+      position: relative;
+      top: -2px;
+    }
+
+    &:hover {
+      &::after {
+        @include di-variant("chevron-down-scampi");
+      }
+    }
+  }
+
+  .dso-group-label {
+    color: $grijs-40;
+    font-size: (14 / 16 * 1em);
+    font-weight: 400;
+    margin: 0;
+    padding: ($u1 / 2) $dso-dropdown-hor-padding ($u1 / 4);
+    text-transform: uppercase;
+  }
+
+  ul {
+    margin: 0;
+    padding: 0;
+
+    &:not(:last-child) {
+      border-bottom: 1px solid $grijs-10;
+      margin-bottom: 11px;
+      padding-bottom: 11px;
+    }
+  }
+
+  li {
+    list-style: none;
+
+    a {
+      @include anchor-reverse();
+
+      clear: both;
+      color: $grijs-90;
+      display: block;
+      font-weight: 400;
+      line-height: 1.5;
+      padding: 3px $dso-dropdown-hor-padding;
+      text-decoration: none;
+      white-space: nowrap; /* stylelint-disable-line declaration-property-value-disallowed-list */
+
+      &:hover,
+      &:focus {
+        background-color: $grasgroen;
+        border-color: $grasgroen;
+        color: $wit;
+        text-decoration: none;
+      }
+    }
+  }
+}
+
+@mixin dropdown-menu_open() {
+  > .dso-primary,
+  > .btn-primary {
+    &::after {
+      @include di-variant("chevron-up-wit");
+    }
+  }
+
+  > .dso-secondary,
+  > .btn-default {
+    &::after {
+      @include di-variant("chevron-up");
+    }
+
+    &:hover::after,
+    &:focus::after {
+      @include di-variant("chevron-up-wit");
+    }
+  }
+
+  > .dso-tertiary,
+  > .btn-link {
+    &::after {
+      @include di-variant("chevron-up");
+    }
+
+    &:hover::after {
+      @include di-variant("chevron-up-scampi");
+    }
+  }
+}
+
+@mixin dropdown-menu_checkable() {
+  a,
+  .dso-group-label {
+    padding-left: $u5;
+  }
+
+  li {
+    a,
+    .dso-group-label {
+      padding-left: $u5;
+    }
+
+    &.dso-checked {
+      a {
+        &:not(:focus) {
+          background-color: $grasgroen;
+          border-color: $grasgroen;
+          color: $wit;
+        }
+
+        &::before {
+          @include di("check-wit", $u3);
+
+          content: "";
+          display: block;
+          float: left;
+          margin-left: -$u4;
+          margin-right: $u1;
+        }
+      }
+    }
+  }
+}
+
+@mixin dropdown_right() {
```

Comment:
@daverhelst: De naamgeving van deze mixins is **altijd** `filename_mixin-name` en `filename` en `mixin-name` zijn altijd kebabcase

#enanderstaart

Reply:
@daverhelst Heb je 'm gelezen?

Reply:
Zeker, maar deze is toch al aangepast naar `dropdown-menu_align-right()`?

Reply:
Klopt, maar als ik #enanderstaart declare is het wel eerlijk dat je 't ook hebt gezien.. ;)

Want bij de volgende verkeerde naamgeving van 'n namespaced mixin... 🍰